components/content/ProseA.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <NuxtLink
+    :to="props.href"
+    class="no-underline hover:no-underline inline-flex justify-center items-center"
```

Comment:
We want an external link to open in a new tab so you'll need:
`:target = "isExternalLink ? '_blank' : '_self'"`

Reply:
Nice catch! I've added this in